src/common/components/Navbar/NavbarData.tsx

```diff
@@ -14,25 +15,25 @@ const NavbarData = () => {
   const getNavbarData = () => {
     return NAVBAR_DUMMY_DATA.map((option) => (
       <div className="text-center" key={option.id}>
-        <a
-          href={option.link as string}
-          className={`cursor-pointer text-sm font-medium ${
+        <Link
+          to={option.link as string}
```

Comment:
Please remove "as string". option.link should be of type string by declaration

Reply:
Fixed this, the problem was caused because link was missing at id:4 object and was declared string | undefined